### PR TITLE
Removes CMake/common dependency from build system

### PR DESCRIPTION
**Pull request by NadirRoGue**
https://bbpteam.epfl.ch/project/issues/browse/BRAYNS-244

- Removes CMake/common submodules
- Rewrite full build system using plain CMake
- Remove all git submodules, replace them for on-demand fetched contents
- Removes cross dependency between braynsCommon and braynsEngine

### CODE REVIEW

**Comment by NadirRoGue** [comment]
retest this please

**Comment by NadirRoGue** [comment]
retest this please

File: engines/ospray/OSPRayModel.cpp [hunk 1]
```diff
@@ -144,12 +144,13 @@ OSPGeometry& OSPRayModel::_createGeometry(GeometryMap& map,
                                           const char* name)
 {
     auto& geometry = map[materialId];
-    if (geometry)
-    {
-        ospRemoveGeometry(_primaryModel, geometry);
-        ospRelease(geometry);
-    }
-    geometry = ospNewGeometry(name);
+    // if (geometry)
```
Comment (Adrien4193) [inline engines/ospray/OSPRayModel.cpp:147]: We leave this commented or we can remove it ?

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -1,51 +1,82 @@
...
 # This file is part of Brayns <https://github.com/BlueBrain/Brayns>
 
-set(TEST_LIBRARIES brayns braynsIO braynsManipulators)
+include(CTest)
+enable_testing()
 
 configure_file(paths.h.in ${PROJECT_BINARY_DIR}/tests/paths.h)
 
-list(APPEND EXCLUDE_FROM_TESTS myPlugin.cpp)
+#list(APPEND CMAKE_MODULE_PATH ${OSPRAY_CMAKE_ROOT})
```
Comment (Adrien4193) [inline tests/CMakeLists.txt:13]: We leave that commented or we can remove it ?

**Review by Adrien4193 — APPROVED** [review]

File: engines/ospray/OSPRayModel.cpp (hunk 1 above)
Comment (NadirRoGue) [reply engines/ospray/OSPRayModel.cpp]: Removed, thanks!

File: tests/CMakeLists.txt (hunk 2 above)
Comment (NadirRoGue) [reply tests/CMakeLists.txt]: Removed, thanks!

**Review by Adrien4193 — APPROVED** [review]

**Pull request merged**